docs/api-types/service-import.md

```diff
@@ -0,0 +1,59 @@
+# ServiceImport API Reference
+
+## Introduction
+
+`ServiceImport` is a resource referring to a Service outside the cluster. The primary use-case of ServiceImport is
+to pair with [`ServiceExport`](service-export.md) resource, defined in the other clusters.
+
+Just like Services, ServiceImports can be a backend reference of HTTPRoutes. Along with the cluster's own Services
+(and ServiceImports from even more clusters), you can distribute the traffic across multiple VPCs and clusters.
+
+Internally, `ServiceImport` is simply a reference to a VPC Lattice target group. Even if a target group is not created
+by ServiceExport, the controller can still recognize it as long as its name matches the ServiceImport.
+This way, you can also send traffic to target groups with EC2 or Lambda targets.
```

Comment:
> the controller can still recognize it as long as its name matches the ServiceImport

I think "name match" is important to highlight. It should be exact name match with TG? I think it also can be TG-id and ARN, right?

Also curious for ServiceExport - do we create TG without random suffix?

Reply:
oh yes, I almost forgot that we have changed the way of identification. I think importing non-exported target group won't work anymore at this point. I will remove that use case instead.